apps/web/src/views/Info/components/CurrencyLogo/index.tsx

```diff
@@ -24,16 +24,11 @@ export const CurrencyLogo: React.FC<
     chainName?: MultiChainName
   }>
 > = ({ address, size = '24px', chainName = 'BSC', ...rest }) => {
-  const src = useMemo(() => {
-    return getTokenLogoURL(new Token(multiChainId[chainName], address as Address, 18, ''))
-  }, [address, chainName])
```

Comment:
You should not remove the image for trustwallet.
We need both, trustwallet image and ourself image url.